lib/Unicode/Internal/Bits.hs

```diff
@@ -49,3 +54,30 @@ lookupBit64 addr# (I# index#) = W# (word## `and#` bitMask##) /= 0
 #endif
     bitIndex# = index# `andI#` fbs#
     bitMask## = 1## `uncheckedShiftL#` bitIndex#
+
+{-| @lookupIntN addr index@ looks up for the @index@-th @8@-bits word in
+the bitmap starting at @addr@, then convert it to an Int.
+
+The caller must make sure that:
+
+* @ceiling (addr + (n * 8))@ is legally accessible @Word8@.
+-}
+lookupIntN
+  :: Addr# -- ^ Bitmap address
+  -> Int   -- ^ Word index
+  -> Int   -- ^ Resulting word as 'Int'
+lookupIntN addr# (I# index#) = I# (word2Int# word##)
+  where
+#if MIN_VERSION_base(4,16,0)
+#ifdef WORDS_BIGENDIAN
```

Comment:
I don't think this is correct: there is nothing to swap in a single byte. The situation is different from `lookupBit64`, where we index a whole word and must swap bytes inside. 

Since you created tests, you can add big-endian CI config similar to https://github.com/composewell/unicode-transforms/pull/75 to check that things work well.

Reply:
I overlooked this one, sorry. I fixed it by reverting the corresponding commit. 

However I do not know how to implement the CI:
- The tests just compare functions to their equivalent in `Data.Char`.
- In order to work, we need a version of GHC using the same version of Unicode than this package. 
- However, currently there is only GHC 9.2.1, which is not available in Ubuntu 20.04 nor using [hvr’s ppa](https://launchpad.net/~hvr/+archive/ubuntu/ghc).

Reply:
Right, you want be able to test big-endian GHC 9.2.1, because there is simply no bindist at the moment. It should be fine as is.

Reply:
Should I adapt your CI using same GHC from Ubuntu, meaning tests will be skipped? Or is better to spare CI till we get the ghc and create an issue to keep track ?

Reply:
Nevermind, GHC 9.2.1 bindist for big endian is not yet happening in the nearest years.